### ISSUE TITLE
Slow driver initialization on startup

### ISSUE THREAD

**Issue by lunabug58** [a1xd/rawaccel#75]
I'm using RawAccel to slow down my mouse speed while retaining the improvement to latency high CPI brings.

The problem lies in that on a fresh boot of the system, it often takes upwards of 2 _minutes_ for the sensitivity multiplier to kick in. My short term workaround has been to simply start the configuration GUI up and hit apply.

Attached is an image of "Apply settings at startup" checked, to show I haven't missed that.
![image](https://user-images.githubusercontent.com/58349791/108024187-d90b2880-706f-11eb-9d58-8ad014a73580.png)

OS: Windows 10 Pro x64 20H2

**Comment by GitLinker** [comment]
I haven't run into this issue, as the driver seems to be loaded by the time I log into windows. Potentially slower computer, running on a hard drive instead of a solid-state drive. It's less common for people to run hard drives as their primary os storage, but not impossible for people on a budget for their pc.

> My short-term workaround has been to simply start the configuration GUI up and hit apply.
I'm gonna assume that means the driver itself seems to be loading perfectly fine?
Run the writer.exe on boot instead?

How I'm specifically doing it:
Windows Key + R
shell:startup
then I made a bat file
`cd Raw Accel .\writer.exe settings.json`
and I moved my raw accel folder

Tell me if that works for you or not.
It's ghetto, but after reading issue #55 . I did this to negate that issue.
